tag/TagV@.cc.lua

```diff
@@ -0,0 +1,276 @@
+		{{ $tagName := index $args 0 }}
+		{{ $tagContent := slice $args 1 | joinStr " " }}
+		{{ if reFind $safeName $tagName }}
+			{{ $data := sdict "Name" $tagName }}
+			{{ template "getTag" $data }}
+			{{ with $data.Tag }}
+				{{ dbSet 0 .Key $tagContent }}
+				Successfully edited the content of the tag `{{ $tagName }}`.
+			{{ else }}
+				Sorry, that tag does not exist!
+			{{ end }}
+		{{ else }}
+			That tag does not exist!
+		{{ end }}
+ 
+	{{ else if and $isCreator (reFind `addalias(es)?` $cmd) (ge (len $args) 2) }}
+		{{ $tagName := index $args 0 }}
+		{{ $aliases := slice $args 1 }}
+		{{ $valid := true }}
+		{{ $key := printf "tg.|%s|" $tagName }}
+		{{- range $k, $ := $aliases -}}
+			{{ if not (reFind $safeName .) }}
+				{{ $valid = false }}
+			{{ else if $k }}
+				{{ $key = joinStr "" $key "|" (lower .) }}
+			{{ else }}
+				{{ $key = joinStr "" $key (lower .) }}
+			{{ end }}
+		{{ end }}
+		{{ if and (reFind $safeName $tagName) $valid }}
+			{{ $data := sdict "Name" $tagName }}
+			{{ template "getTag" $data }}
+			{{ with $data.Tag }}
+				{{ dbDelByID .UserID .ID }}
+				{{ $newKey := joinStr "" $key "|" }}
+				{{ if gt (len $newKey) 256 }}
+					Sorry, that alias was too long. Try again.
+				{{ else }}
+					{{ dbSet 0 (joinStr "" $key "|") .Value }}
+					Successfully added {{ len $aliases }} aliases to the tag `{{ $tagName }}`!
+				{{ end }}
+			{{ else }}
+				Sorry, that tag does not exist.
+			{{ end }}
+		{{ else }}
+			Sorry, some aliases provided were not valid! Try again.
+		{{ end }}
+ 
+	{{ else if and $isCreator (reFind `(del(ete)?|remove)alias(es)?` $cmd) (ge (len $args) 2) }}
+		{{ $tagName := index $args 0 }}
+		{{ $toRemove := slice $args 1 | joinStr " " | lower }}
+		{{ $data := sdict "Name" $tagName }}
+		{{ template "getTag" $data }}
+		{{ with $data.Tag }}
+			{{ $aliases := split (slice .Key 4 (sub (len .Key) 1)) "|" }}
+			{{ $tagName := printf "tg." }}
+			{{ if eq (len $aliases) 1 }}
+				Sorry, you cannot remove an alias from a tag with only 1 alias.
+			{{ else }}
+				{{ range $aliases }}
+					{{ if ne $toRemove . }}
+						{{ $tagName = joinStr "" $tagName "|" . }}
+					{{ end }}
+				{{ end }}
+				{{ $tagName = joinStr "" $tagName "|" }}
+				{{ dbDelByID .UserID .ID }}
+				{{ dbSet 0 $tagName .Value }}
+				Successfully removed alias!
+			{{ end }}
+		{{ else }}
+			That tag does not exist.
+		{{ end }}
+ 
+	{{ else if reFind `list|show|all` $cmd }}
+		{{ $page := 1 }}
+		{{ if eq (len $args) 1 }} {{ with reFind `^\d+$` (index $args 0) }} {{ $page = toInt . }} {{ end }} {{ end }}
+		{{ $skip := mult (sub $page 1) 10 }}
+		{{ $tags := dbTopEntries "tg.|%|" 10 $skip }}
+		{{ if not (len $tags) }}
+			There were no tags, go make some!
+		{{ else }}
+			{{ $display := "" }}
+			{{ $total := len $tags }}
+			{{- range $k, $ := $tags -}}
+				{{ $tagName := title (index (split (slice .Key 4 (sub (len .Key) 1)) "|") 0) }}
```

Comment:
I made the list a code block with css, so the name is blue, aliases yellow, and date is in red. I'm having trouble getting rid of the tag name among the aliases, although I may leave it so that if a tag has no name that spot is still filled.